packages/protocol/test-sol/governance/network/EpochRewards.t.sol

```diff
@@ -5,18 +5,18 @@ import "celo-foundry/Test.sol";
 import "@celo-contracts/common/Registry.sol";
 import "@celo-contracts/common/Freezer.sol";
 
-import { MockElection } from "@celo-contracts/governance/test/MockElection.sol";
-import { EpochRewardsTest } from "@celo-contracts/governance/test/EpochRewardsTest.sol";
+import "@celo-contracts/governance/test/MockElection.sol";
+import { EpochRewardsMock } from "@celo-contracts/governance/test/EpochRewardsMock.sol";
```

Comment:
Should this also be moved to `@test-sol/...`?

Reply:
Ditto as AddressSortedLinkedListWithMedianMock